guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,47 @@
+[id="migrating-to-a-new-server_{context}"]
+
+= Migrating to a new server
+
+.Prerequisites
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
```

Comment:
Please move this section to a different file using a level 1 heading & then include it in an assembly.

Reply:
This guide doesn't use any other assemblies, so I've moved the section to a procedure/reference and the team will address use of assemblies at a later date.

Reply:
OK, not using assemblies is fine if you directly include the files in a `master.adoc` file. You must not include files in other files, like including a procedure in another procedure. Also, you cannot have two headings in one procedure.